packages/fxa-payments-server/src/components/TermsAndPrivacy/index.tsx

```diff
@@ -27,6 +28,10 @@ export const TermsAndPrivacy = ({ plan }: TermsAndPrivacyProps) => {
     ? productDetailsFromPlan(plan, navigatorLanguages)
     : DEFAULT_PRODUCT_DETAILS;
```

Comment:
I don't think we should ever rely on the defaults going forward.  If the Stripe metadata are missing or incorrect, we'll fix it there.

Reply:
I guess this is the importance of updating the docs to specify these keys in the metadata to add when adding a product. I wonder if there's a way to list certain metadata as required, so that products in production can't be added without them?

Reply:
I think refusing to show the subscription creation page if the product is "mis-configured" would be a good feature.  I'll file an issue.